.github/workflows/build-linux.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - replace_trimesh
```

Comment:
I guess we don't need this

Reply:
Darn, I thought I took that out, maybe that was on replace_trimesh and not replace_trimesh_alt.